src/GeneralStateTestsFiller/stExtCodeHash/extCodeHashSubcallOOGFiller.yml

```diff
@@ -0,0 +1,149 @@
+# create contract A in a subcall. go OOG in a subcall (revert happens) check EXTCODEHASH of A (in upper call)
+---
+extCodeHashSubcallOOG:
+  _info:
+    comment: "create contract A in a subcall. go OOG in a subcall (revert happens) check EXTCODEHASH of A (in upper call)"
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: "10000000"
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    1000000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        {
+          (CALLCODE 350000 (CALLDATALOAD 0) 0 0 0 0 32)
+          (SSTORE 1 (EXTCODEHASH (MLOAD 0)))
+          (SSTORE 2 (EXTCODESIZE (MLOAD 0)))
+          (EXTCODECOPY (MLOAD 0) 0 0 32)
+          (SSTORE 3 (MLOAD 0))
+          (SSTORE 4 (CALLCODE 50000 (MLOAD 0) 0 0 0 0 0))
+        }
+      nonce: '0'
+      storage: {}
+    2000000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (CALL 150000 0xa000000000000000000000000000000000000000 0 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    2100000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (CALLCODE 150000 0xa000000000000000000000000000000000000000 0 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    2200000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (DELEGATECALL 150000 0xa000000000000000000000000000000000000000 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    3000000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (CALL 150000 0xa100000000000000000000000000000000000000 0 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    3100000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (CALLCODE 250000 0xa100000000000000000000000000000000000000 0 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    3200000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { (DELEGATECALL 150000 0xa100000000000000000000000000000000000000 0 0 0 32) (RETURN 0 32)}
+      nonce: '0'
+      storage: {}
+    #create dynamic contract not going out of gas
+    a000000000000000000000000000000000000000:
+      balance: '1000000000000000000'
+      code: |
+        { 
+          (MSTORE 0
+            (CREATE2 0 0
+              (lll
+              {
+                (MSTORE 0 0x6020602055)
+                (RETURN 27 5)
+              }
+              0)
+            0))
+           (RETURN 0 32)
+           (STOP)
+        }
+      nonce: '0'
```

Comment:
Yes, this new format is actually really beautiful and nice to read!

Reply:
right but hash checking is awful